Library/Homebrew/os/linux/elf.rb

```diff
@@ -172,6 +201,14 @@ def needed_libraries_using_readelf(path)
     end
   end
 
+  def patcher
```

Comment:
```suggestion
  def patcher
    return unless HOMEBREW_PATCHELF_RB
    
```

Reply:
patchelf_patcher is only when HOMEBREW_PATCHELF_RB is enabled, do we need this extra check ?

Reply:
Yes, want to ensure we cannot attempt to require when `HOMEBREW_PATCHELF_RB` is not set (if it's called elsewhere in future).

Reply:
@MikeMcQuaid  if this is for development time safety, would it not be better to exit with a helpful message?